landavailability/webclient/views.py

```diff
@@ -18,7 +18,7 @@ class SearchView(TemplateView):
     def get(self, request, *args, **kwargs):
         context = self.get_context_data(**kwargs)
 
-        postcode = request.GET.get('location')
+        postcode = request.GET.get('location') or request.GET.get('postcode')
```

Comment:
why both postcode and location assigned to postcode? What do we pass from the UI?